### DISCUSSION TITLE
Will the integration eliminate cells?

### DISCUSSION THREAD

**Discussion by hyjforesight** [satijalab/seurat#5021] [Q&A]
Hello all,
I'm a little confused with the integration method.
Here is my understanding. When we have query data and reference data, the integration method will create an anchor between 2 similar cell clusters of the query and reference. This anchor is created based on the similarity of feature expression patterns of these 2 clusters, like a common feature subset between them. So these 2 similar cell clusters will be UMAPed at the same place in the integrated UMAP, which is called removing batch effects.
My question is that whether this integration method will remove cells from the clusters after integration? For instance, Cluster 1 in query has 1000 cells. Cluster 2 in reference has 1500 cells. After integration, only 800 cells from query and 1000 cells from reference are UMAPed at the same site.
Thanks!
Best,
YJ

**Comment by hyjforesight** [comment]
I tested by cell.num<-table(Idents(object)). Integreation does not remove cells.

**Comment by timoast** [comment]
Integration will not remove cells, see here for a full explanation of the method: https://doi.org/10.1016/j.cell.2019.05.031
